### PR TITLE
Add SalesAgility to swag list.

### PR DESCRIPTION
**Pull request by Taminoful**
SalesAgility gives away a mug for each contributor who submitted at
least one merged pull request to one of their mentioned repositories.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,15 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: [Join the Discord](https://discord.gg/python).
 - **Notes**:  See this page for their [post](https://www.reddit.com/r/Python/comments/dbj0vh/python_discord_hacktoberfest_2019/).
 
+### S
+
+#### **SalesAgility**
+
+- **Swag**: SuiteCRM contributor mug
+- **Requirements**: Create at least 1 merged PR to [the SuiteCRM, SuiteDocs or SuitePortal repositories](https://github.com/api-platform) during October.
```
Comment (rafaeelaudibert) [inline README.md]: I think you left the wrong link here

Comment (Taminoful) [reply README.md]: Thank you for mentioning! Fixed it! 😃

**Comment by crweiner** [comment]
Wow, thanks for finding a new company to add to the list @Taminoful!! I appreciate it!! If you could please make sure that SalesAgility is alphabetized properly under the **1 Merged PR** header, I'd appreciate that, and then we can get this merged. :+1:

**Comment by Taminoful** [comment]
@crweiner Just fixed that. 🚀

**Comment by crweiner** [comment]
Marvelous, thank you so much! Let's get this live! :smile: 

thumbs up

**Pull request merged**